deploy/operator/setup_lso.sh

```diff
@@ -21,6 +21,33 @@ function install_lso() {
   retry -- oc annotate project openshift-local-storage openshift.io/node-selector='' --overwrite=true
 
   catalog_source_name="redhat-operators"
+
+  OC_VERSION_MAJOR_MINOR=$(oc version -o json | jq --raw-output '.openshiftVersion' | cut -d'.' -f1-2)
+  if [[ ${OC_VERSION_MAJOR_MINOR} == "4.12" ]]; then
+      # LSO has not been published to the 4.12 redhat-operators catalog, so
```

Comment:
I knew that sounded familiar b26b8d43dd6678ca4ac4333ffd389a680d7ab740
 :sob: 
 
 Missed its birthday by a few days!

Reply:
We have it every single release. I for example copied this one from `MGMT-11011: use 4.10 index for LSO installation in 4.11 (#4060)`